### ISSUE TITLE
Initalize first image using data-image-list

### ISSUE THREAD

**Issue by v1t3** [scaleflex/js-cloudimage-360-view#33]
Hi!
When initialize div with param data-image-list i'm getting an empty first image. But if I try to change slide, images are loading and scrolling properly.
Adding param data-filename for first image don't take effects.

Here are codesandbox where issue reproduced https://codesandbox.io/s/js-cloudimage-360-view-controls-ln3bp?file=/index.html

p.s. thnx for great plugin!

**Comment by akito6776** [comment]
I just found out about this awesome javascript library and also ran into this same issue during implementation. (which is that the library acts weird when "data-image-list" is used)

After looking through the code, it looks like this is caused by a simple oversight and can be easily fixed.

The issue is with the "preloadImages()" function, and more specifically, the part of the if loop that deals with the usage of "data-image-list". It calls "addImage()", but fails to provide the last parameter for the function, which is an "index" number.

Here's the relevant section of the code with the fix, which is the sole addition of the "imgIndex" variable. Basically, 3 lines to edit. (I'm showing the non-minified code here for readability. You probably want to minify it back before using it.)
```
preloadImages(amount, src, lazyload, lazySelector, container, responsive, ciParams) {
  if (this.imageList) {
    try {
      const images = JSON.parse(this.imageList);
      var imgIndex = 0;

      this.amount = images.length;
      images.forEach(src => {
        const folder = /(http(s?)):\/\//gi.test(src) ? '' : this.folder;
        const resultSrc = this.getSrc(responsive, container, folder, src, ciParams);

        this.addImage(resultSrc, lazyload, lazySelector, imgIndex);
        imgIndex++;
      });
    } catch (error) {
      console.error(`Wrong format in image-list attribute: ${error.message}`);
    }
  } else {
    [...new Array(amount)].map((_item, index) => {
      const nextZeroFilledIndex = pad(index + 1, this.indexZeroBase);
      const resultSrc = src.replace('{index}', nextZeroFilledIndex);
      this.addImage(resultSrc, lazyload, lazySelector, index);
    });
  }
}
```
Hopefully, this can help others while we wait for an official fix from the dev.

**Comment by dzmitry-stramavus** [comment]
@v1t3 , @akito6776 

fixed in **2.4.1**

@akito6776 , thx for debugging!
